libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc

```diff
@@ -5,7 +5,7 @@
 -------------------------------------------------------------------------------
 processors:
 - add_host_metadata:
-    netinfo.enabled: false
+    netinfo.enabled: true
```

Comment:
Do we still need this with func defaultConfig() has `NetInfoEnabled: true`?

Reply:
Not really needed, but it was also not needed before, I can remove it. I don't have an opinion on this, so as you prefer.

Reply:
Cool, I just want to make sure it's consistent with the rest of the files in Beats. How about removing it?

Reply:
Removed :+1:

Reply:
Thanks!